src/libraries/Native/build-native.sh

```diff
@@ -307,66 +286,30 @@ while :; do
         staticliblink|-staticliblink)
             __StaticLibLink=1
             ;;
-        -portable|-portable)
+        portable|-portable)
             # Portable native components are only supported on Linux
             if [ "$__HostOS" == "Linux" ]; then
                 __PortableBuild=1
             fi
             ;;
-        --clang*)
+        clang*|-clang*)
```

Comment:
Can you keep the `--clang` too please? People may rely on that form in their local build stuff.

Reply:
Included `--clang*`.
IMO, we should deprecate this one-off `--` option for consistency.